networkx/algorithms/bridges.py

```diff
@@ -295,8 +213,8 @@ def bridges_exist(G):
     exist in a given network.
     """
 
-    results = all_bridges(G)
```

Comment:
Hey, your latest build looks like it's going to pass!

One last thing from me: you can leverage the fact that `bridges()` is a generator to make this more efficient. Right now, your code constructs the entire list. You can actually dodge that—a lot like how you short-circuit `local_bridges()` using `first_match`. Here's how I'd do it:

```python
    try:
        next(bridges(G))
    except StopIteration:
        return False
    return True
```
If you do something like this, be sure to update the docstring accordingly. Does this approach make sense?